Werkgroep API design rules/naamgeving.md

```diff
@@ -0,0 +1,56 @@
+# Naamgeving
+## Generieke naamconventies
+### Speciale tekens
+- Diakrieten normaliseren naar basis-karakter
+- Symbolen/leestekens (met uitzondering van een koppelteken) weglaten
+ 
+## Resource namen en path segments
+Omdat resources dingen beschrijven (in plaats van acties), worden resource namen als zelfstandig naamwoorden beschreven. Enkelvoudige resources worden in enkelvoud geschreven, collection resources in meervoud:
+(etc--kan voortborduren op 3.1 Resources van de API Design Rules).
+
+In het geval van samengestelde resource namen of path segments, wordt de kebab-case casing conventie gehanteerd.
+
+### Voorbeelden:
+- `/gebouw`
+- `/vergunningen`
+- `/organisatie-codes`
+ 
+### Niet doen:
+- Gebruik geen operaties zoals gebruikelijk is bij RPC-style API's (bv /getResource).
+- Refereer niet in de naam dat iets een API is (bv /resource_api).
+- Gebruik geen bestands extensies in de resource namen (bv /resource.json).
+    - Het gebruik van Content Negotiation via de Accept header heeft de voorkeur, al zijn API aanbieders vrij om convenience endpoints aan te bieden op basis van extensies.
+- Voorkom waar mogelijk het gebruik van afkortingen voor concepten, zoals `/coords` ipv `/coordinaten`. Een uitzondering hierop is het gebruik van breed bekende afkortingen als kwalificatie in veldnamen zoals `bagNummeraanduiding`. 
+- Gebruik geen samengestelde namen voor geneste objecten (bv /adres_straat).
+    - Gebruik hiervoor een sub-object 'adres' met daarin een 'straat' parameter.
+
+## Resource bodies
+- Gebruik JSON of JSON gebaseerde documenttypen (bv JSON+HAL)
```

Comment:
Gebruik van JSON heeft geen relatie tot naamgeving, al zijn veel naming conventies wel gekoppeld aan een formaat. Ik zou in dit geval deze 2 bullets samenvoegen tot:
> Gebruik camelCase conventie voor JSON property-namen. Voor andere formaten zou evt een andere conventie kunnen gelden.

Reply:
Eens @joostfarla , echter staat er al een principe rondom JSON formaat in de extensies (API-22) en staat er al iets over in de ADR extensies. Aangezien het niet direct in de scope van naamgeving valt, heb ik de tekst zo laten staan (al vind ik zelf API-22 nog wat summier beschreven). Ook omdat ik niet weet of die in het verleden al is gereviewd, afgestemd en vastgesteld..